CWE/history/countries/TNZ - Tanganyika-Zanzibar.txt

```diff
@@ -51,7 +51,7 @@ schools = army_tech_school
 1992.1.1 = {
 	capital = 2036
 	religion = secularism
-	government = democracy
+	government = presidential_dictatorship
```

Comment:
Socialist democracy might work better, though my knowledge of them is kind of fuzzy - and also it might lead to wonky things with spheres.